data/successors/successor ships.txt

```diff
@@ -839,6 +843,8 @@ ship "Seiitej"
 	gun -5 -86
 	gun 5 -86
 	gun 10 -78
+	pylon -10 -78
+	pylon 10 -78
```

Comment:
The Seiitej does not have enough weapons capacity to equip two Sjejan kinetic lances (90/120), so I think it would make the most sense from an in-universe perspective to have a single pylon on the ship's midline rather than two offset ones.